src/Symfony/Component/JsonStreamer/Tests/Fixtures/Model/DummyWithArray.php

```diff
@@ -4,7 +4,7 @@
 
 class DummyWithArray
 {
-    /** @var ClassicDummy[] */
+    /** @var list<ClassicDummy> */
```

Comment:
I am not sure if it is actually a good idea to do this change or if we shouldn’t introduce additional `DummyWithList` and `DummyWithNestedList` fixtures instead to cover both cases.